addons/openebs/1.6.0/host-preflight.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: HostPreflight
+metadata:
+  name: kurl-builtin
+spec:
+  collectors:
+  - blockDevices:
+      # cstor is enabled and not upgrade
+      exclude: '{{kurl and (and .Installer.Spec.OpenEBS.Version .Installer.Spec.OpenEBS.IsCstorEnabled) (not .IsUpgrade) | not }}'
```

Comment:
Is there a simpler way to write this exclude?

Reply:
maybe, but this is copied (and cropped) from the original